libcontainer/container_linux.go

```diff
@@ -2321,6 +2321,18 @@ func (c *Container) bootstrapData(cloneFlags uintptr, nsMaps map[configs.Namespa
 		})
 	}
 
+	// write boottime and monotonic time ns offsets.
+	if c.config.Namespaces.Contains(configs.NEWTIME) && c.config.TimeOffsets != nil {
+		var offsetSpec bytes.Buffer
```

Comment:
I think it's an overkill to use `bytes.Buffer` here.

Something like
```go
spec := fmt.Sprintf("%s %d %d\n", clock, offset.Secs, offset.Nanosecs)
...
Value: []byte(spec)
```

should be enough.

Yet better (but less readable) would be to do
```go
// Same as fmt.Sprintf("%s %d %d\n", clock, offset.Secs, offset.Nanosecs) but faster.
spec := clock + " " + strconv.Itoa(offset.Secs) + " " strconv.Itoa(int64(offset.Nanesecs)) + "\n"
```

but I guess sprintf is fine, too.

Reply:
Ah, sorry, I totally missed the fact that we are adding multiple strings together. In this case bytes.Buffer is fine, please ignore the above comment!